README.md

```diff
@@ -157,6 +170,9 @@ All you need is Docker and git (optional).
      -e IMAGE_STREAMER_SSL_ENABLED=true \
      puppet-oneview puppet apply image_streamer/deployment_plan.pp --debug --trace
    ```
+
+#### Using Docker Hub Image
```

Comment:
It should be near pull docker image from Docker Hub

Reply:
Done